package.json

```diff
@@ -31,5 +31,8 @@
     "lint-staged": "^15.2.0",
     "prettier": "^3.1.1",
     "yorkie": "^2.0.0"
+  },
+  "dependencies": {
+    "@eslint/eslintrc": "^3.1.0"
   }
```

Comment:
This should be in `packages/migrate-config/package.json`, not in the top-level package.json?